packages/sign-client/src/constants/engine.ts

```diff
@@ -91,12 +91,12 @@ export const ENGINE_RPC_OPTS: EngineTypes.RpcOptsMap = {
   },
   wc_sessionPing: {
     req: {
-      ttl: THIRTY_SECONDS,
+      ttl: ONE_DAY,
```

Comment:
Seems to be going off-spec? https://specs.walletconnect.com/2.0/specs/clients/sign/rpc-methods#wc_sessionping